### DISCUSSION TITLE
oversample.exe being flagged as a virus

### DISCUSSION THREAD

**Discussion by sison54** [nothings/stb#1481] [Q&A]
Hello,

My IT department is saying that our firewall is flagging tests/oversample/oversample.exe as a virus.

I was just wondering if anyone has any thoughts on this topic or offer any advise for some reassurance I can give them that this is in fact not a virus and just a false positive.

Thanks

**Comment by nothings** [comment]
Given that it's never been touched since it was first checked in 9 years ago, unless github itself is secretly adding viruses to files it must be a false positive.

**Comment by sison54** [comment]
Thank you, that was my conclusion as well but they are being overly cautious.

**Reply by metiscus** [reply]
I just saw this today but I think I may know why it was flagged. I threw the exe file into Jotti's malware scan and it shows that the exe is upx packed. Many virus scanners automatically flag upx packed binary files as containing viruses anymore even when they really dont. 

PE32 executable (GUI) Intel 80386, for MS Windows, UPX compressed

Here is the results from my Jotti scan showing the file is clean. https://virusscan.jotti.org/en-US/filescanjob/nu6jx1efd6

**Comment by sison54** [comment]
Just to follow up. The firewall company did come back and say the file was clean and this was in fact a false positive.
